src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java

```diff
@@ -148,6 +160,8 @@ public static void closeBrowser() {
     }
   }
 
+  private static LinkedList<String> parentWindows = new LinkedList<>();
```

Comment:
Move it back to the class fields please.

Reply:
Done